lib/db/schema/patch-040-041.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS emailBounces (
+  email VARCHAR(255) NOT NULL,
+  bounceType TINYINT UNSIGNED NOT NULL,
+  bounceSubType TINYINT UNSIGNED NOT NULL,
+  createdAt BIGINT UNSIGNED NOT NULL,
+  PRIMARY KEY(email, createdAt)
+) ENGINE=InnoDB CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+CREATE PROCEDURE `createEmailBounce_1` (
+  IN inEmail VARCHAR(255),
+  IN inBounceType TINYINT UNSIGNED,
+  IN inBounceSubType TINYINT UNSIGNED,
+  IN inCreatedAt BIGINT UNSIGNED
+)
+BEGIN
+  INSERT INTO emailBounces(
+    email,
+    bounceType,
+    bounceSubType,
+    createdAt
+  )
+  VALUES(
+    inEmail,
+    inBounceType,
+    inBounceSubType,
+    inCreatedAt
+  );
+END;
+
+CREATE PROCEDURE `fetchEmailBounces_1` (
+  IN `inEmail` VARCHAR(255)
+)
+BEGIN
+  SELECT
+      email,
+      bounceType,
+      bounceSubType,
+      createdAt
+  FROM emailBounces
+  WHERE email = inEmail
+  ORDER BY createdAt DESC
+  LIMIT 20;
+END;
+
+UPDATE dbMetadata SET value = '40' WHERE name = 'schema-patch-level';
```

Comment:
needs update to 41 here , just like the change in https://github.com/mozilla/fxa-auth-db-mysql/pull/199/files#diff-217a4921cc0a8155a99a66420c71397d